### ISSUE TITLE
Unable to farm drops for Pokemon TCG Live

### ISSUE THREAD

**Issue by DovydasTEDS** [DevilXD/TwitchDropsMiner#402]
```
14:46:36: Fatal error encountered:
14:46:36: 
14:46:36: Traceback (most recent call last):
14:46:36:   File "main.py", line 158, in main
14:46:36:   File "twitch.py", line 764, in run
14:46:36:   File "twitch.py", line 913, in _run
14:46:36:   File "twitch.py", line 1691, in get_live_streams
14:46:36:   File "twitch.py", line 1548, in gql_request
14:46:36: exceptions.MinerException: GQL error: [{'message': 'service error', 'path': ['game']}]
14:46:36: 
14:46:36: Exiting...
14:46:36: 
14:46:36: Application Terminated.
14:46:36: Close the window to exit the application.
```

I think this may be because the game is called 'Pokémon TCG Live'
The letter 'é' might be causing the issue.
Works without that game added to my list.

**Comment by DevilXD** [comment]
This is a duplicate of an already closed issue: #347. I think you aren't using the latest version of the miner. Please provide the version of the miner you're using, you can find it in the title bar of the application window. It should be either v15 or v16, with some additional letters and numbers too it. A picture/screenshot will do as well.

**Comment by DevilXD** [comment]
Closing because of no response. I'm assuming just using the latest version has solved the problem.
